package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "blackbaud-skyux-lib-help",
-  "version": "1.0.0",
-  "description": "Single-page-application for skyux-lib-help",
+  "name": "BB-Help",
```

Comment:
The name is what people will type when they require or import your library. `@blackbaud/help`. (we already have `@blackbaud/help-client`)